Wikipedia/Code/ReadingListsController.swift

```diff
@@ -90,6 +90,7 @@ public enum ReadingListError: Error, Equatable {
 public class ReadingListsController: NSObject {
     @objc public static let readingListsServerDidConfirmSyncIsEnabledForAccountNotification = NSNotification.Name("WMFReadingListsServerDidConfirmSyncIsEnabledForAccount")
     @objc public static let readingListsServerDidConfirmSyncIsEnabledForAccountIsSyncEnabledKey = NSNotification.Name("isSyncEnabledForAccount")
+    @objc public static let readingListsServerDidConfirmSyncIsEnabledForAccountWasSyncEnabledOnDeviceKey = NSNotification.Name("wasSyncEnabledOnDevice")
```

Comment:
It's a little hard to tease these out. Usually we have one "is" or "was" per key and these have "...is...is..." and "is...was" respectively:

`readingListsServerDidConfirmSyncIsEnabledForAccountIsSyncEnabledKey`
`readingListsServerDidConfirmSyncIsEnabledForAccountWasSyncEnabledOnDeviceKey`

... any thoughts on distilling these down a bit?

Reply:
It's fine if these are actually named accurately too - just double-checking :)

Reply:
yeah, they are named accurately but if you can think of better names, I'll be happy to update these, in `readingListsServerDidConfirmSyncIsEnabledForAccountWasSyncEnabledOnDeviceKey`  there's `was` instead of `is` cause we want to know if sync **was** ever enabled on this device, in `readingListsServerDidConfirmSyncIsEnabledForAccountIsSyncEnabledKey ` it's `is`, instead of `was` cause it's supposed to reflect the current sync state for the account. I guess we could re-name `readingListsServerDidConfirmSyncIsEnabledForAccountIsSyncEnabledKey` to `readingListsServerDidConfirmSyncWasEnabledForAccountWasSyncEnabledKey` if you think that'd be better?

Reply:
Ah thanks! Yes, I think your "was...was" idea would make it better 👍